src/IconAccessibilityTagHelper.cs

```diff
@@ -1,7 +1,7 @@
-﻿namespace Tailwind.Heroicons;
+namespace Tailwind.Heroicons;
 
 /// <summary>
-/// Tag helper that sets the <code>aria-hidden</code> or <code>role</code> attribute based on if <code>aria-label</code> or <code>aria-labeledby</code> are set.
+/// Tag helper that sets the <c>aria-hidden</c> or <c>role</c> attribute based on if <c>aria-label</c> or <c>aria-labeledby</c> are set.
```

Comment:
Currently these don't show correctly in the VS html editor tooltip, but an issue has been reported about it.

ref: https://developercommunity.visualstudio.com/t/Tag-helper-tooltip-doesnt-support-code-/10167092